nixos/modules/installer/cd-dvd/iso-image.nix

```diff
@@ -37,6 +38,28 @@ let
     )
   ;
 
+  signFile = file: let
+    name = baseNameOf file;
+    signDrv = pkgs.runCommand "${name}-signed" { nativeBuildInputs = [ pkgs.buildPackages.sbsigntool ]; } ''
```

Comment:
Unrelated to this PR: When would I need to use pkgs.buildPackages in nativeBuildInputs? I always thought that should be automatically be done.

Reply:
The splicing that allows (e.g.) sbsigntool to mean something different in `nativeBuildInputs` than it does in `buildInputs` is done by `callPackage` iirc, so you need to specify buildPackages when you're not in a `callPackage`d context (if you want cross to work).

https://nixos.org/manual/nixpkgs/stable/#ssec-cross-dependency-implementation